src/mcts/search.h

```diff
@@ -328,6 +328,12 @@ class SearchWorker {
   const bool moves_left_support_;
   IterationStats iteration_stats_;
   StoppersHints latest_time_manager_hints_;
+  // Intermediate array to store values when processing policy.
+  // According to a lichess developer post:
+  // https://lichess.org/blog/Wqa7GiAAAOIpBLoY/
+  //     developer-update-275-improved-game-compression
+  // there are never more than 256 valid legal moves in any legal position.
+  float intermediate_[256];
```

Comment:
No need to have it as class member, it's fine to have this on stack inside function.
Without initializer (`= {}`), it's not taking any time to initialize (as it will be left uninitalized).

Even better, use `std::array<float, 256>;` instead. It still won't be initialized.

Reply:
Looks good to me:
===========================
Total time (ms) : 340519
Nodes searched  : 70133528
Nodes/second    : 205960